lib/game_box_web/live/arena_live.ex

```diff
@@ -304,10 +318,37 @@ defmodule GameBoxWeb.ArenaLive do
     {:noreply, assign(socket, version: version)}
   end
 
+  # Each time the players are updated, if the process
+  # receiving the message belongs to the host, we want to send a check in 5 seconds that
```

Comment:
Why after 5 seconds instead of now?

Reply:
If you refresh the page you technically get removed until the page remounts so I didn't want to have this flash on and off every time someone did that or if they accidentally navigated away but made it back in a relatively short amount of time.